Includes/survival-games.xml

```diff
@@ -682,5 +723,5 @@
         <spawn>spawner</spawn>
     </filter>
 </mobs>
-<timelock>on</timelock>
-</map>
+<timelock>${timelock}</timelock>
```

Comment:
I can't remember why timelock was in the include in the first place, but I think it would be better to leave this up to the map by adding the exact setting they want instead of making a constant just for this.

Reply:
👍